src/main/java/uk/gov/hmcts/reform/coh/controller/AnswerController.java

```diff
@@ -87,9 +86,11 @@ public ResponseEntity createAnswer(UriComponentsBuilder uriBuilder, @PathVariabl
         try {
 
             Optional<Question> optionalQuestion = questionService.retrieveQuestionById(questionId);
+
             // If a question exists, then it must be in the issued state to be answered
             if (!optionalQuestion.isPresent()
-                    || !optionalQuestion.get().getQuestionState().getState().equals(QuestionStates.ISSUED.getStateName())) {
+                    || !optionalQuestion.get().getQuestionState().getState().equals(QuestionStates.ISSUED_PENDING.getStateName())
```

Comment:
If it's issue pending, then the question hasn't been sent

Reply:
When I asked you if an appellant can answer a question in issued pending state you said they can :(

Reply:
I did? I don't remember. Apologies if I did.

Reply:
For the answer cucumber tests to work the question must be question_issued. For this to happen the question issued event must be processing by the scheduler and update the question_issued_pending to question_issued after forwarding the notification. At the moment this functionality does not exist.

Reply:
So it's up to you whether to wait until someone has developed the functionality for the question_issued_pending state to be updated question_issued state. Or to merge it now and i'll add fix the state later